README.md

```diff
@@ -1,68 +1,212 @@
 # veritable-ui
 
-UI for `Veritable` utilizing [TSOA](https://tsoa-community.github.io/docs/getting-started.html), [HTMX](https://htmx.org/) and [@kitajs/html](https://www.npmjs.com/package/@kitajs/html) for JSX templating.
+A user interface for `Veritable` that allows to manage connections across supply chain and perform queries in relation to the supply chain. Utilizing [TSOA](https://tsoa-community.github.io/docs/getting-started.html), [HTMX](https://htmx.org/) and [@kitajs/html](https://www.npmjs.com/package/@kitajs/html) for JSX templates.
 
-## Setup
+#### Table of Contents
 
-To install dependencies:
+- [Setup](#setup)
+- [Getting Started](#getting-started)
+- [Development Mode/Local dev](#development-mode)
+- [Environment Variables](#environment-variables)
+- [Testing](#testing)
 
-```bash
+## Setup
+veritable-ui depends on a few external services:
+- [veritable-cloudagent](https://github.com/digicatapult/veritable-cloudagent) - APIs for managing connections, credentials and messages.
+- `keycloak` - runs as a docker image for each node that handles authentication and users.
+- `docker` - for orchestrating multiple nodes and running other dependencies such as database (PostgreSQL) along with node specific environment variables
```

Comment:
I wouldn't list docker as a dependent service as it doesn't need docker really. I would however have `postgresql`

Reply:
It has been removed